packages/database/prisma/schema.prisma

```diff
@@ -8,7 +8,213 @@ datasource db {
 }
 
 model Study {
-  id        Int      @id @default(autoincrement())
-  createdAt DateTime @default(now())
-  updatedAt DateTime @updatedAt
+  id                         Int                       @id
+  name                       String
+  //cpmsId                     String
+  status                     String
+  //statusId                   String
+  recordStatus               String
+  route                      String
+  irasId                     String?
+  protocolReferenceNumber    String?
+  sampleSize                 Int?
+  chiefInvestigatorFirstName String?
+  chiefInvestigatorLastName  String?
+  managingSpeciality         String
+  plannedOpeningDate         DateTime?
+  plannedClosureDate         DateTime?
+  actualOpeningDate          DateTime?
+  actualClosureDate          DateTime?
+  assessments                Assessment[]
+  organisations              StudyOrganisation[]
+  funders                    StudyFunder[]
+  evaluationCategory         StudyEvaluationCategory[]
+  createdAt                  DateTime                  @default(now())
+  updatedAt                  DateTime                  @updatedAt
+  isDeleted                  Boolean?                  @default(false)
+}
+
+model Organisation {
+  id            Int                 @id @default(autoincrement())
+  name          String              @unique
+  rtsIdentifier String              @unique
+  createdAt     DateTime            @default(now())
+  updatedAt     DateTime            @updatedAt
+  isDeleted     Boolean?            @default(false)
+  users         UserOrganisation[]
```

Comment:
What purpose do these navigation properties play on the 'Organisation' entity?

Reply:
These are the opposite relations for the `createdBy` and `updatedBy` fields on `UserOrganisation`. Without them the Prisma schema fails validation.

Reply:
OK - I can't see us ever needing to navigate from the organisation to one of those entities though. Rather, the other way around. Bit weird that Prisma errors!